fendermint/actors/blobs/src/state.rs

```diff
@@ -1010,20 +1040,21 @@ impl State {
             // We can ignore later finalizations, even if they are failed.
             return Ok(());
         }
-        let group =
-            blob.subscribers
-                .get_mut(&subscriber.to_string())
-                .ok_or(ActorError::forbidden(format!(
-                    "subscriber {} is not subscribed to blob {}",
-                    subscriber, hash
-                )))?;
+        let mut subscribers = blob.subscribers.hamt(store)?;
+        let mut group = subscribers
+            .get(&subscriber)
```

Comment:
maybe you missed a question mark here? instead of `unwrap`

Reply:
Good call, this is fixed.